mkdocs.yml

```diff
@@ -1,6 +1,7 @@
 site_name: Shoutrrr
 site_url: https://containrrr.dev/shoutrrr/
 repo_url: https://github.com/containrrr/shoutrrr/
+edit_uri: edit/main/docs/index.md
```

Comment:
```suggestion
edit_uri: edit/main/docs/
```

Reply:
Not sure. I did the same commit on watchtower repo and I think it works there.

Reply:
In any case, I applied the suggestion.